integration-tests/environments/react-native/package.json

```diff
@@ -18,9 +18,8 @@
     "app-importer": "realm-app-importer serve ../../realm-apps",
     "metro": "react-native start --reset-cache",
     "runner": "node harness/runner.js",
-    "prepare": "npm run install-local && npm run pod-install",
-    "install-local": "npm_config_ignore_scripts=true install-local",
-    "pod-install": "cd ios && pod install"
+    "prepare": "npm run install-local && pod-install",
+    "install-local": "npm_config_ignore_scripts=true install-local"
```

Comment:
Using the `pod-install` package makes it easier to run these on a non-mac machine.